pkg/cvo/updatepayload.go

```diff
@@ -295,6 +280,8 @@ func (r *payloadRetriever) pruneJobs(ctx context.Context, retain int) error {
 // `mkdir -p <target dir> && mv <default release manifest dir> <target release manifests dir>`
```

Comment:
Update, now three commands

Reply:
Good catch.  No need to echo what folks can read a few lines down in this internal function, so I've just dropped this from the docstring with 624f7fd0 -> d70b3c9e.